workflows/chatbot/inference/generate.py

```diff
@@ -210,16 +212,9 @@ def __call__(
         return False
 
 
-def max_input_len(input_text_length):
```

Comment:
Why change this？ Please rebase your code.

Reply:
OK, I'll rebase my code.